server/.env

```diff
@@ -1 +1,4 @@
-PORT = 2999
\ No newline at end of file
+PORT = 2999
+CLIENT_URL = http://127.0.0.1:5173/
+STREAM_API_KEY= efb7xtpvjqsx
```

Comment:
well, this is not a good practise but for now lets keep it

we will use [github secrets](https://github.com/Azure/actions-workflow-samples/blob/master/assets/create-secrets-for-GitHub-workflows.md) when our CD pipeline is ready

Reply:
okay. That would be good